zookeeper-server/src/test/java/org/apache/zookeeper/common/FileChangeWatcherTest.java

```diff
@@ -31,36 +31,25 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import org.apache.commons.io.FileUtils;
 import org.apache.zookeeper.ZKTestCase;
-import org.apache.zookeeper.test.ClientBase;
-import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class FileChangeWatcherTest extends ZKTestCase {
 
-    private static File tempDir;
-    private static File tempFile;
+    @TempDir
+    static File tempDir;
+    static File tempFile;
```

Comment:
There's actually a lot of places where you can make these private. Here's some more, but I won't comment on every one in the PR.

```suggestion
    private static File tempDir;
    private static File tempFile;
```

Reply:
Thanks for the comments, But Junit does not let me make that private. Pls see
```
org.junit.jupiter.api.extension.ExtensionConfigurationException: @TempDir field [private static java.io.File org.apache.zookeeper.client.ZKClientConfigTest.testDataDir] must not be private.
```